Languages/ChineseSimplified/Keyed/ChineseSimplified.xml

```diff
@@ -1,23 +1,23 @@
-<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <LanguageData>
 
-	<!-- Mod Settings -->
+  <!-- Mod Settings -->
 
-	<HMTB_EnableMod>使用搬运工来建造</HMTB_EnableMod>
+  <HMTB_EnableMod>在建造中使用搬运工</HMTB_EnableMod>
 
-	<HMTB_OpportunisticMode>蓝图或工地搜索物品范围</HMTB_OpportunisticMode>
+  <HMTB_OpportunisticMode>蓝图/框架的搜索半径</HMTB_OpportunisticMode>
 
-	<!-- Tooltips -->
+  <!-- Tooltips -->
 
-	<HMTB_Tooltip_EnableMod>所有指定搬运工作的居民同样也会把材料运送到工地。</HMTB_Tooltip_EnableMod>
+  <HMTB_Tooltip_EnableMod>任何设置为搬运工的殖民者都会為建造工作运送材料。</HMTB_Tooltip_EnableMod>
```

Comment:
“為”only used in Traditional Chinese

Reply:
This is even more confusing, most Chinese input method in Simplified Chinese mode will not prompt the that character since “为" is a commonly used character。to type “為”,an ordinary people have to switch input method into Traditional Chinese mode or after checking more than a dozen pages since it is not used in Simplified Chinese, for this specific character.